hide/comp/cdb/Cell.hx

```diff
@@ -30,6 +30,20 @@ class Cell extends Component {
 				e.stopPropagation();
 				@:privateAccess line.table.toggleList(this);
 			});
+		case TFile:
+			element.on("drop", function(e : js.jquery.Event) {
+				var e : js.html.DragEvent = untyped e.originalEvent;
+				if (e.dataTransfer.files.length > 0) {
+					e.preventDefault();
+					e.stopPropagation();
+					setValue(ide.makeRelative(untyped e.dataTransfer.files.item(0).path));
+					refresh();
+				}
+			});
+			if ( canEdit() )

```

Comment:
Shouldn't the drop listener be in `if ( canEdit() )` too btw?

Reply:
I'm actually unsure. @ncannasse I'm not sure how `canEdit()` determines editability. Should TFile check for it, or be editable always? And if it have to be enforced, shouldn't TScript also check for `canEdit()`?

Reply:
Yes canEdit() should be taken into account. It's a new feature to allow partial edition so is not yet done everywhere.